analyzer/src/main/kotlin/MavenSupport.kt

```diff
@@ -125,6 +133,16 @@ class MavenSupport(localRepositoryManagerConverter: (LocalRepositoryManager) ->
             }
 
     fun requestRemoteArtifact(artifact: Artifact, repositories: List<RemoteRepository>): RemoteArtifact {
+        val cacheKey = artifact.toString()
```

Comment:
Use `String.fileSystemEncode()` instead?

Reply:
The DiskLruCache only allows keys matching "[a-z0-9_-]{1,64}" which is stricter than fileSystemEncode().